components/adblock_rust_ffi/src/lib.rs

```diff
@@ -3,9 +3,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-use adblock::engine::Engine;
+use adblock::engine::{Engine, EngineDebugInfo};
 use adblock::lists::FilterListMetadata;
 use adblock::resources::{MimeType, Resource, ResourceType};
+use adblock::filters::regex_manager::RegexManagerDiscardPolicy;
```

Comment:
Heads up about https://github.com/brave/adblock-rust/commit/031882d3ca0b4f31d9404a5b8d2af924705aaee4 - this will need to be `adblock::regex_manager::RegexManagerDiscardPolicy`

Reply:
Done